### ISSUE TITLE
High negative score with Vina

### ISSUE THREAD

**Issue by cpayal** [ccsb-scripps/AutoDock-Vina#168]
I suspect this could be related to the issue raised earlier #68 , but I still see the problem:

Version: AutoDock Vina v1.2.3
OS : Linux (Ubuntu 2022)
 usage : 
`vina --receptor rigid_rec.pdbqt --flex flex_rec.pdbqt --batch input_pdbqts/Z1*.pdbqt --config dock.config --exhaustiveness 8 --dir  flex_results/ --cpu 8 --verbosity 2 >> flex_verbose1.log`

Vina output pdbqt: 
```
MODEL 1
REMARK VINA RESULT: -46687.173      0.000      0.000
REMARK INTER + INTRA:     4658906.646
REMARK INTER:                 -12.948
REMARK INTRA:             4658919.595
REMARK UNBOUND:           4719240.480
REMARK  Name = Z653910660
```

However, shouldn't the intermolecular interactions be printed as results ? A related output pdbqt is attached [here](https://github.com/cpayal/bugs/tree/main/autodock-vina).

Thanks

**Comment by diogomart** [comment]
Thank you for reporting. Could you share the input files to reproduce? If not, could you try without the `--flex` to see if it still happens?

**Comment by diogomart** [comment]
Probably fixed by c1217c1ac7f5cfdd5931eba01f187dd967969c16.
